### PR TITLE
Creating releases...

### PR DESCRIPTION
**Pull request by bobbyrullo**
...how to do it?

### CODE REVIEW

File: Documentation/releases.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+  ```console
+  $ git clone git@github.com:coreos/dex.git
+  $ cd dex
+  ```
+
+1. Tag with the release name:
+
+   ```console
+   git tag -s v0.4.0
```
Comment (sym3tri) [inline Documentation/releases.md]: We should really be signing the releases.

Comment (ericchiang) [inline Documentation/releases.md]: Other than docker containers there often aren't assets to sign.

Comment (sym3tri) [inline Documentation/releases.md]: I mean sign the tag. 
```
-s, --sign
           Make a GPG-signed tag, using the default e-mail address's key.
```

Comment (sym3tri) [inline Documentation/releases.md]: But I didn't realize the `-s` was already there, sorry.

Comment (sym3tri) [inline Documentation/releases.md]: We should probably make a note that the releaser needs upload their public GPG key to github tho.

**Comment by ericchiang** [comment]
lgtm

**Pull request merged**